### PR TITLE
Rework reporting and merge flow

### PR DESCRIPTION
**Pull request by Tritium-VLK**
Failed first test.

### CODE REVIEW

**Comment by Tritium-VLK** [comment]
ok @SHAKOTN I think this is all really working more (def a squash merge).

#352 is an example merge of 4269-W69 into this branch that I plan to close once we agree it is gud/working.

What a pain, but now I understand.

**Comment by Tritium-VLK** [comment]
Spoke too soon.  So close.

**Comment by Tritium-VLK** [comment]
ok #354

File: .github/workflows/process_every_payload.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Process BIP PR
```
Comment (SHAKOTN) [inline .github/workflows/process_every_payload.yaml:1]: why need this?

**Review by SHAKOTN — APPROVED** [review]

File: .github/workflows/process_every_payload.yaml (hunk 1 above)
Comment (Tritium-VLK) [reply .github/workflows/process_every_payload.yaml:1]: The naame or the action.

The action is needed to trigger the reports when a PR is opened (under the on conditions listed in this file).

The Name probs isn't needed, but it makes it easier to look at.  What this does is runs on PR and does bip processing.

**Pull request merged**